man/man1/p11kmip.1.in

```diff
@@ -1,6 +1,6 @@
-.TH P11KMIP 1 "August 2023" "@PACKAGE_VERSION@" "openCryptoki"
+.TH P11KMIP 1 "August 2023" "3.23" "openCryptoki"
```

Comment:
Please leave this as `@PACKAGE_VERSION@` this will then insert the current package version during build.